inc/cleantalk-public-integrations.php

```diff
@@ -1038,24 +1040,29 @@ function ct_preprocess_comment($comment)
         }
     }
 
+    $sender_info = array(
+        'sender_url'      => @$comment['comment_author_url'],
```

Comment:
can I refuse @? Why @?

Reply:
It's an old fix. We can't just remove it. If we want to remove it we need to test the removal separately.